pkg/web/templates/components/transaction_rejection.html

```diff
@@ -8,7 +8,7 @@ <h3 class="font-bold text-xl">Reject Transaction</h3>
       </button>
     </div>
     <div class="my-4">
-      <form id="transaction-reject-form" method="dialog" hx-post="/v1/transactions/{{ .ID }}/reject" hx-ext="json-enc" hx-swap="none">
+      <form id="transaction-reject-form" method="dialog" hx-post="/v1/transactions/{{ .ID }}/reject" hx-ext="json-enc" hx-swap="none" hx-indicator="#spinner">
```

Comment:
`hx-indicator` was added here to ensure the spinning loader appears while a request is pending. I attempted to hide the button text but will need to handle that with JS, so decided to handle via a separate story so that all buttons in submit forms may be updated at once.

Reply:
the button should probably be disabled right? multiple clicks on the button could create weirdness on the backend, which I've already managed to do.

Reply:
I've noticed that some submit buttons have a disabled state while others don't. Ensuring consistency across all will be included as part of the follow on story as well.